src/components/editor/DiffEditor.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { DiffEditor } from "@monaco-editor/react";
+
+function MyDiffEditor() {
+  const originalCode = `
+  function add(a, b) {
+    return a + b;
+  }`;
+
+  const modifiedCode = `
+  function add(a: i32, b: i32): i32 {
+    return a + b;
+  }`;
+
+  return (
+    <DiffEditor
+      height="6.9rem"
+      width="98%"
+      theme="vs-dark"
+      language="javascript"
+      original={originalCode}
+      modified={modifiedCode}
+      options={{
+        renderSideBySide: true,
+        readOnly: true,
+      }}
+    />
+  );
+}
+
+export default MyDiffEditor;
```

Comment:
통일성을 위해 함수 정의 부분에서 

```suggestion
export default function MyDiffEditor() {
```

로 수정해주시면 좋을 것 같습니다😀

Reply:
확인했고 바로 반영했습니다!